Automattic/wp-calypso
# i18n: Unifying languages

#### Changes proposed in this Pull Request

* Add notice in language picker modal when non-magnificent locale is selected
* Replace hard coded languages data with `languages-meta.json` downloaded from i18n-deploy
* Add setting for using default locale as fallback for languages with incomplete translations.

![image](https://user-images.githubusercontent.com/2722412/86332349-1642b780-bc53-11ea-8443-509951168b10.png)

#### Testing instructions

* Boot Calypso (`yarn start`)
* Confirm that `languages-meta.json` is downloaded successfully in `client/languages/`. Make sure it's downloading the JSON file from D46739-code.
* Go to [Account Settings](http://calypso.localhost:3000/me/account), select non-magnificent language (e.g. Croatian) and confirm that language notice is being rendered.
* Update interface language and enable `Display interface in Engllish` option. Confirm interface renders in English after saving the settings.


#### Related
D45711-code
D46739-code
